### PR TITLE
Updated css/html and started implementation of submarine

### PR DESCRIPTION
**Pull request by ramosjos**
Updated the css and html to have buttons for the submarine and for the movement of the fleet. Also started the implementation of the submarine. As of now it is just a size 5 ship, and works fine. Still need to make it so it isnt size 5 and can be placed underwater.

### CODE REVIEW

**Comment by ramosjos** [comment]
I forgot to add that I also added some tests for the submarine.

**Comment by ramosjos** [comment]
Please review and merge if you guys like the changes I have made.

File: src/main/java/assets/css/custom.css [hunk 1]
```diff
@@ -223,9 +224,10 @@ top:100px;
     left:100px;
 position:fixed;
 left:100px;
-top:200px;
+top:300px;
 }
-.batmov{
+
+.submov{
```
Comment (JordanNg) [inline src/main/java/assets/css/custom.css]: Nice CSS for the submarine

File: src/main/java/assets/css/custom.css [hunk 2]
```diff
@@ -242,6 +244,45 @@ top:200px;
...
+  border-radius: 15px;
+  box-shadow: 0 9px #999;
+    position:fixed;
+    left:100px;
+position:fixed;
+left:100px;
+top:100px;
+}
+
+.fleetmov{
```
Comment (JordanNg) [inline src/main/java/assets/css/custom.css]: Nice job adding CSS for the fleets movement

File: src/main/java/assets/game.js [hunk 3]
```diff
@@ -86,7 +93,7 @@ function cellClick() {
             game = data;
             redrawGrid();
             placedShips++;
-            if (placedShips == 3) {
+            if (placedShips == 4) {
```
Comment (JordanNg) [inline src/main/java/assets/game.js]: YES!

File: src/main/java/cs361/battleships/models/Ship.java [hunk 4]
```diff
@@ -86,6 +89,21 @@ public void place(char col, int row, boolean isVertical) {
 						occupiedSquares.add(new Square(row, (char) (col + i)));
 					}
 				}
+			} else if (size == 5){
```
Comment (JordanNg) [inline src/main/java/cs361/battleships/models/Ship.java]: Nice job implementing the submarine! From what I can tell it is a submarine is just of length 5. I can try and see if I am able to make it into the weird shape that is being required.

File: src/test/java/cs361/battleships/models/ShipTest.java [hunk 5]
```diff
@@ -70,6 +70,33 @@ public void testPlaceBattleshipHorizontaly() {
 
     @Test
     public void testPlaceBattleshipVertically() {
+        Ship minesweeper = new Ship("SUBMARINE");
```
Comment (JordanNg) [inline src/test/java/cs361/battleships/models/ShipTest.java]: Nice tests!

**Pull request merged**